pkg/webhooks/server.go

```diff
@@ -139,8 +151,10 @@ func NewWebhookServer(
 		pLister:                   pInformer.Lister(),
 		pSynced:                   pInformer.Informer().HasSynced,
 		rbLister:                  rbInformer.Lister(),
+		rLister:                   rInformer.Lister(),
 		rbSynced:                  rbInformer.Informer().HasSynced,
 		crbLister:                 crbInformer.Lister(),
+		crLister:                  crInformer.Lister(),
```

Comment:
Need to add `rInformer.Informer().HasSynced` and `crInformer.Informer().HasSynced`, to let the cache sync at least once before controller starts. 
Also need to add `sync` func to: https://github.com/nirmata/kyverno/blob/8cf5cd70fec8cf4e7ec304eafa73c722c1e1f74e/pkg/webhooks/server.go#L520

Reply:
ok